app/ModelValidator/SiteValidator.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: macintosh
+ * Date: 6/20/16
+ * Time: 9:10 AM
+ */
+
+namespace App\ModelValidator;
+
+
+use App\Libs\CdsApiDriver;
+use App\Libs\ColdValidator;
+
+class SiteValidator extends BaseValidator
+{
+    public function _name($name) {
+        if(strlen($name) == 0) {
+            $this->bindError(__FUNCTION__, 'Name can not be empty');
+            return false;
+        }
+
+//        if(!(ColdValidator::instance()->isJapanese($name) || preg_match('/^[0-9a-zA-Z._# ]+$/', $name))) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.relipa.blog/coding_rules#rule_key=php%3AS125)